eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -997,21 +1029,8 @@ electBasicCommittee ::
     , GovPurposeId 'CommitteePurpose era
     )
 electBasicCommittee = do
-  logEntry "Setting up PParams and DRep"
-  modifyPParams $ \pp ->
-    pp
-      & ppDRepVotingThresholdsL
-        %~ ( \dvt ->
-              dvt
-                { dvtCommitteeNormal = 1 %! 1
-                , dvtCommitteeNoConfidence = 1 %! 2
-                , dvtUpdateToConstitution = 1 %! 2
-                }
```

Comment:
Looks like we lost all of those threshold in the setup of every test that used to `electBasicCommittee`? Are they set somewhere else that I am missing?

Reply:
Indeed! I forgot that these were set here!  Will make sure they are set, thanks!